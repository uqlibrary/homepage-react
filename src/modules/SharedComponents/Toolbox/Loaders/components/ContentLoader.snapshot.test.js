import ContentLoader from './ContentLoader';

function setup(testProps) {
    const props = { ...testProps };
    return getElement(ContentLoader, props);
}

describe('ContentLoader snapshots tests', () => {
    it('renders loader', () => {
        const wrapper = setup({ message: 'Waiting to load...' });
        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    });
});
