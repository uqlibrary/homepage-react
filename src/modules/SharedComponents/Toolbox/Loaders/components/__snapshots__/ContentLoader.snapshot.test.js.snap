// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentLoader snapshots tests renders loader 1`] = `
<StandardPage>
  <ForwardRef(Grid)
    container={true}
  >
    <ForwardRef(Grid)
      item={true}
      xs={12}
    >
      <WithStyles(InlineLoader)
        message="Waiting to load..."
      />
    </ForwardRef(Grid)>
  </ForwardRef(Grid)>
</StandardPage>
`;
